docs/cli/edgedb_migration.rst

```diff
@@ -24,13 +24,69 @@ description. It's also possible to split the schema definition across
 multiple ``.esdl`` files. The migration tools will read all of them
 and treat them as a single SDL document.
 
+Here is typical file tree::
+
+    ❯ tree dbschema
```

Comment:
Inconsistent use of shell prefixes, maybe stick to `$` since they are used everywhere else in documentation?

Reply:
Fixed